README.md

```diff
@@ -5,46 +5,58 @@ This readme assumes that you have the following programmes installed:
 - IDE e.g. VS Code
 - MongoDB Compass
 - Node (version 19.0.1)
+- Git
 
-This ReadMe has been made in a .md format so it can be opened in your IDE previewer for easier reading.
+**PLEASE NOTE:**
+
+This ReadMe was written using the built-in terminal in VS Code. It was tested using this terminal on both MacOS and WindowsOS. 
 
 --- 
 ## Setup & Install
-Navigate to the folder where you want the application to be stored on your computer   
+
+1. Open a terminal on MacOS or PowerShell on Windows.  
+
+2. Navigate to the folder where you want the application to be stored on your computer.  
 > i.e. `$ cd Desktop/Webapp`  
 
-Clone the repo into this folder from this link --> (https://github.com/OrrionB/UniWebapp.git) 
+3. Clone the repo into this folder from this link --> (https://github.com/OrrionB/UniWebapp.git) 
 > i.e `$ git clone https://github.com/OrrionB/UniWebapp.git`  
   
-Install the dependencies  
-1. Open 2 terminal windows.
-2. In the first termnial window navigate to the folder where you installed the webapp and run the command `npm install`.  
+ 4. Open the folder in VS Code. 
+  
+4. Install the dependencies  
+- Open 2 terminals in VSCode  
+- In the first termnial window navigate to the folder where you installed the webapp and run the command `npm install`.  
 > i.e `$ cd Desktop/Webapp`  
 >
 >`$ npm install`  
 
-3. In the second terminal window, navigate to the **front-end** folder inside the repo and again run the command `npm install`.  
+- In the second terminal window, navigate to the **front-end** folder inside the repo and again run the command `npm install`.  
 > i.e `$ cd Desktop/Webapp/front-end`  
 >  
 > `$ npm install`
 ---
 
 
 ## Creating the Database
+
+Open MongoDB Compass 
+
 Create a Database on `localhost:27017` in MongoDB Compass called **`Webapp_Project`**.  
+
 In **Webapp_Project** please create the following collections:  
 1. **`FunFacts`**
 2. **`JobTitles`**
 3. **`Names`**
 4. **`Races`**  
   
-  Into these collections, you can import the corresponding JSON file from **`database-jsons`**.
+  Into these collections, you can import the corresponding JSON file from the **`database-jsons`** folder that was included in the repo.
 
 ---
 
 ## Starting the Application  
 1. Make sure you have followed the above steps for setup and creation of the database.
-2. In the terminal window that is in the directory where you cloned the app, run the command `nodemon`.
+2. In the terminal window that is in the directory where you cloned the app, run the command `node index.js`.
```

Comment:
this isn't how I start it. Also there are two index.js files. I go into front end and run npm start